lib/elixometer.ex

```diff
@@ -255,17 +262,18 @@ defmodule Elixometer do
   If the value of the `:reset_seconds` option is greater than zero, the counter will be reset
   automatically at the specified interval.
   """
+  @spec update_counter(bitstring, float, [reset_seconds: nil | pos_integer]) :: :ok
```

Comment:
This must be `integer` as it uses `:ets.update_counter` that only works on integers.